### PR TITLE
Fix import setting module

### PR DESCRIPTION
**Pull request by aprilahijriyan**

### CODE REVIEW

**Review by Nimond — APPROVED** [review]

**Comment by Nimond** [comment]
Thanks a lot!

File: saq/worker.py [hunk 1]
```diff
@@ -280,6 +281,7 @@ def import_settings(settings):
 
 
 def start(settings, web=False, extra_web_settings=None, port=8080):
+    sys.path.insert(0, os.getcwd())  # fix: https://github.com/tobymao/saq/issues/26
```
Comment (barakalon) [inline saq/worker.py:284]: This feels risky to me. We don't actually know what directory saq is being invoked from.

_If_ we add something like this, it should probably be in https://github.com/tobymao/saq/blob/master/saq/__main__.py

**Comment by aprilahijriyan** [comment]
I agree with @barakalon suggestion. How about you @tobymao ?

**Comment by barakalon** [comment]
To be clear, I'm leaning heavily _away_ from this.

**Comment by tobymao** [comment]
@aprilahijriyan as i said it’s best if you fix your app or add a python path, this is how arq works as well fyi

**Pull request closed without merging**